.github/workflows/release-please.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+      - main
+name: release-please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          release-type: node
+          package-name: generator-eslint
+      - uses: actions/checkout@v3
+        if: ${{ steps.release.outputs.release_created }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+          registry-url: https://registry.npmjs.org
+        if: ${{ steps.release.outputs.release_created }}
+      - run: npm ci
+        if: ${{ steps.release.outputs.release_created }}
+      - run: npm test
```

Comment:
Currently, we run one final round of tests as part of releases. I'm assuming we want to keep doing that. In theory, the full test matrix will have already passed on the release PR in order for us to merge it, so this single run is adding little value. It also introduces some risk: if the release-please step creates the release on GitHub but a flaky test fails in this step, the `npm publish` step won't run. Then if we re-run this workflow, release-please will see that the release was already created, skipping the rest of this workflow. We'd have to manually delete the release from GitHub before re-running.

Reply:
I think it's okay to skip testing here.

Reply:
It's gone!